README.md

```diff
@@ -0,0 +1,90 @@
+# Instructions to create a new plugin
```

Comment:
This is missing a link to https://github.com/halkyonio/operator-framework/tree/master/plugins

Reply:
Fixed